env_file_app_travis

```diff
@@ -39,7 +39,7 @@ GREYNOISE_API_KEY=test
 INTELX_API_KEY=test
 WHOISXMLAPI_KEY=test
 UNPAC_ME_API_KEY=test
-URLSCAN_API_KEY=
+URLSCAN_API_KEY=test
```

Comment:
this should be left blank for the URLscan test to complete successfuly

Reply:
ah, why?

Reply:
The `URLScan_Search` analyzer works without API key but the submit_result requires API key. If we give dummy api key, then search one fails

Reply:
I commented the `submit_result` variant of URLscan for this reason only and that's why the test is failing ig

Reply:
I think that this is because it is not mocked `requests.Session.get`

Reply:
yes, that was the cause. Now it works.